sonar-ws-generator/run.sh

```diff
@@ -3,7 +3,9 @@ set -euo pipefail
 
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 
-cd $SCRIPTPATH
-mvn clean package -Prun-ws-generator
-java -jar target/sonar-ws-generator-*-SNAPSHOT-jar-with-dependencies.jar
-cp -R target/generated-sources/results/org/sonarqube/ws/client/* ../sonar-ws/src/main/java/org/sonarqube/ws/client
+pushd $SCRIPTPATH/..
+pwd
+./gradlew :sonar-ws-generator:build :sonar-ws-generator:fatJar
+popd
+java -jar build/libs/sonar-ws-generator-*-SNAPSHOT-jar-with-dependencies.jar
+cp -R build/generated-sources/results/org/sonarqube/ws/client/* ../sonar-ws/src/main/java/org/sonarqube/ws/client
```

Comment:
Someday I think this script could be replaced by a Gradle task. That would avoid to generate a fat jar to deal with dependencies.